content/en/docs/components/pipelines/v2/cli.md

```diff
@@ -4,119 +4,156 @@ description = "Interact with KFP via the CLI"
+
+* To compile a pipeline definition defined in a Python file, run the following command.
+
+  ```shell
+  kfp dsl compile --py [PATH_TO_INPUT_PYTHON] --output [PATH_TO_OUTPUT_YAML] --function [PIPELINE_NAME]
+  ```
+  
+  For example:
+  
+  ```shell
+  kfp dsl compile --py path/to/pipeline.py --output path/to/output.yaml
+  ```
+  
+  To compile a single pipeline or component from a Python file containing multiple pipeline or component definitions, use the `--function` argument.
+  
+  For example:
+  
+  ```shell
+  kfp dsl compile --py path/to/pipeline.py --output path/to/output.yaml --function my_pipeline
+  ```
+  
+  ```shell
+  kfp dsl compile --py path/to/pipeline.py --output path/to/output.yaml --function my_component
+  ```
+
+* To specify pipeline parameters, use the `--pipeline-parameters` argument and provide the parameters as JSON.
+
+  ```shell
+  kfp dsl compile [PATH_TO_INPUT_PYTHON] --output [PATH_TO_OUTPUT_YAML] --pipeline-parameters [PIPELINE_PARAMETERS_JSON]
+  ```
+
+  For example:
+  
+  ```shell
+  kfp dsl compile --py path/to/pipeline.py --output path/to/output.yaml --pipeline-parameters '{"param1": 2.0, "param2": "my_val"}'
+  ```
+
+### Build containerized Python components
+<!-- TODO: Revisit the links after the refactoring is completed -->
+
+You can author [containerized Python components][containerized-python-components] in the KFP SDK. This lets you use handle more source code with better code organization than the simpler [lightweight Python component][lightweight-python-component] authoring experience.
 
-* For `pipeline`:
-  * `create-version`
-  * `list-versions`
-  * `get-versions`
-  * `delete-versions`
-
-
-## Compile pipelines
-The KFP supports compiling a pipeline defined in a Python file to YAML using the `kfp dsl compile` command:
-
-```shell
-kfp dsl compile --py path/to/pipeline.py --output path/to/output.yaml
-```
+<!-- TODO(GA): remove --pre -->
 
-To compile a pipeline from a Python file containing multiple pipeline definitions or to compile a component, provide a `--function` argument:
+#### Before you begin
 
-```shell
-kfp dsl compile --py path/to/pipeline.py --output path/to/output.yaml --function my_component
-```
+Run the following command to install the KFP SDK with the additional Docker dependency:
 
-The CLI compiler also accepts a `--pipeline-parameters` argument as JSON:
 ```shell
-kfp dsl compile --py path/to/pipeline.py --output path/to/output.yaml --pipeline-parameters '{"param1": 2.0, "param2": "my_val"}'
+pip install --pre kfp[all]
 ```
 
-## Build KFP containerized Python components
-The KFP SDK support authoring [containerized Python components][containerized-python-components], allowing the use of more and better-organized source code than does the simpler [lightweight Python component][lightweight-python-component] authoring experience.
+#### Build a containerized component
```

Comment:
Can this be "Build the component" or something similar? While this section is nested within the "Build containerized Python components" section, I think "containerized component" sounds very close to "custom container component" or, at the very least, leaves some ambiguity about component authoring approach we're discussing.